packages/patterns-react/src/internal/FeatureFlags.js

```diff
@@ -77,3 +77,11 @@ export const DDS_CARDS_WITHOUT_IMAGES =
  */
 export const DDS_USECASES =
   process.env.DDS_USECASES === 'true' || DDS_FLAGS_ALL || false;
+
+/**
+ * This determines if the cards with images will be rendered or not
+ *
+ * @type {string | boolean}
+ */
+export const DDS_CARDS_WITH_IMAGES =
+  process.env.CARD === 'true' || DDS_FLAGS_ALL || false;
```

Comment:
@mkothur should be `process.env.DDS_CARDS_WITH_IMAGES`

Reply:
Yeah I will update

Reply:
Can you also add `DDS_CARDS_WITH_IMAGES` to the `.env.example` file within the patterns-react package?